src/main/java/com/synopsys/integration/alert/provider/polaris/descriptor/PolarisGlobalDescriptorActionApi.java

```diff
@@ -84,9 +98,10 @@ public void testConfig(final TestConfigModel testConfig) throws IntegrationExcep
 
     @Override
     public FieldModel saveConfig(final FieldModel fieldModel) {
-        final Optional<AccessTokenPolarisHttpClient> polarisHttpClient = polarisProperties.createPolarisHttpClientSafely(logger);
+        // FIXME had to remove client check for now so that the task actually starts without restarting alert.
```

Comment:
@bamandel What do we want to do here?

Reply:
This will also be handled in a separate PR